cli/generators/crud/index.js

```diff
@@ -7,6 +7,13 @@ module.exports = class extends Generator {
     super(args, opts);
 
     this.argument('name', { type: String, required: true });
+
+    this.option('api', {
```

Comment:
Here the option is `api`, but the docs say `--no-api`. Maybe this one should be changed?